views/needed.erb

```diff
@@ -25,11 +25,11 @@
 
         <div class="container">
             <ul class="grid-list grid-list--vertically-center source-needed">
-              <% @page.to_find.sort_by { |i| i[:title] }.each do |i| %>
+              <% @page.to_find.sort_by { |i| i.title }.each do |i| %>
```

Comment:
These could all be collapsed now to `sort_by(&:title)`, but the deeper issue is that really it shouldn't be up to the _template_ to do that. As we're doing that in all three cases, it would really make much more sense to push it back into the Page class, and have it guarantee to give them in sorted order. 

Up to you whether to make a new Issue for that, and reference it, or do it within this PR…